src/scripting/lua_ui.cc

```diff
@@ -273,12 +273,12 @@ int LuaPanel::get_descendant_position(lua_State* L) {
 	UI::Panel* cur = (*get_base_user_class<LuaPanel>(L, 2))->panel_;
 
 	Vector2i cp = Vector2i::zero();
-	while (cur && cur != panel_) {
+	while ((cur != nullptr) && cur != panel_) {
```

Comment:
So are the parentheses needed or not? I'm never sure about the precedence of comparison operators, so this is too much to me... :-o